config/Dockerfiles/linchpin/conf/linchpin.conf

```diff
@@ -0,0 +1,296 @@
+#async_timeout = 1000
+
+# the uhash value will still exist, but will not be added to
+# instances or the inventory_path
+#enable_uhash = False
+enable_uhash = True
+
+# in older versions of linchpin (<v1.0.4), a resources folder exists, which
+# dumped the data that is now stored in the RunDB. To disable the resources
+# output, set the value to False.
+#generate_resources = True
+generate_resources = False
+
+# default paths in playbooks
+#
+# lp_path = <src_dir>/linchpin
+# determined in the load_config method of # linchpin.cli.LinchpinCliContext
+
+# Each of the following items controls the path (usually along with the
+# default values below) to the corresponding item.
+
+# In the workspace (generally), this is the location of the layouts and
+# topologies looked up by the PinFile. If either of these change, the
+# value in linchpin/templates must also change.
+#layouts_folder = layouts
+#topologies_folder = topologies
+
+# The relative location for hooks
+#hooks_folder = hooks
+
+# The relative location for provider roles
+#roles_folder = roles
+
+# The relative location for storing inventories
+#inventories_folder = inventories
+
+# The relative location for resources output (deprecated)
+#resources_folder = resources
+
+# The relative location to find schemas (deprecated)
+#schemas_folder = schemas
+
+# The relative location to find playbooks
+#playbooks_folder = provision
+
+# The default path to schemas for validation (deprecated)
+#default_schemas_path = {{ lp_path }}/defaults/%(schemas_folder)s
+
+# The default path to topologies if they aren't in the workspace
+#default_topologies_path = {{ lp_path }}/defaults/%(topologies_folder)s
+
+# The default path to inventory layouts if they aren't in the workspace
+#default_layouts_path = {{ lp_path }}/defaults/%(layouts_folder)s
+
+# The default path for outputting ansible static inventories
+#default_inventories_path = {{ lp_path }}/defaults/%(inventories_folder)s
+
+# The default path to the ansible roles which control the providers
+#default_roles_path = {{ lp_path }}/%(playbooks_folder)s/%(roles_folder)s
+
+# In older versions (<1.2.x), the schema was held here. These schemas are
+# deprecated.
+#schema_v3 = %(default_schemas_path)s/schema_v3.json
+#schema_v4 = %(default_schemas_path)s/schema_v4.json
+
+# The location where default credentials data would exist. This path doesn't
+# automatically exist
+#default_credentials_path = %(default_config_path)s
+
+# If desired, one could overwrite the location of the generated inventory path
+#inventory_path = {{ workspace }}/{{inventories_folder}}/happy.inventory
+inventory_path = /tmp/{{ topology_name }}-{{ uhash }}.inventory
+
+# Libvirt images can be stored almost anywhere (not /tmp).
+# Unprivileged users need not setup sudo to manage a path to which they have rights.
+# The following are specific settings to manage libvirt images and instances
+
+# the location to store generated ssh keys and the like
+#default_ssh_key_path = ~/.ssh
```

Comment:
I wonder if this should be using the workdir/ssh ?  Then the user can pass in their own keys.

Reply:
This line is only used currently in the [libvirt provider](https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/linchpin/provision/roles/libvirt/tasks/provision_libvirt_node.yml#L54). Since this container doesn't support libvirt within it yet.